src/debug_commads.c

```diff
@@ -665,6 +667,27 @@ DEBUG_COMMAND(GCForceBGInvoke) {
   return REDISMODULE_OK;
 }
 
+DEBUG_COMMAND(GCWaitForInvokeAndDrop) {
```

Comment:
Maybe we can just add some field for FT.INFO indicate the time for the next GC run? I wonder if its enough for the test?

Reply:
I think it's not. First of all, by the end of this function, the index is removed from all the globals and we cannot call `FT.INFO` on it. Second, the GC runs after the index is freed and doesn't have any access to its statistics anymore.